requirements.txt

```diff
@@ -0,0 +1,6 @@
+virtualenv
```

Comment:
with the latest changes. basically wanted to test if dynamically installing custom themes would work on heroku or not? could you deploy contents of PR #139 ?